packages/foreman/rubygem-mime-types/rubygem-mime-types.spec

```diff
@@ -0,0 +1,103 @@
+# Generated from mime-types-3.2.2.gem by gem2rpm -*- rpm-spec -*-
```

Comment:
we already have mime-types in the Rails SCL: https://github.com/theforeman/rails-packaging/tree/tfm-ror52/packages/tfm-ror52-rubygem-mime-types